doc/scripts/extract_content.py

```diff
@@ -92,15 +98,17 @@ def get_rst_files(dest, dir):
 
 def main():
 
-    if len(sys.argv) < 3:
-        print("usage: {} <dest> <org dirs>", file=sys.stderr)
-        sys.exit(1)
+    parser = argparse.ArgumentParser(description='Extract documentation content.')
+
+    parser.add_argument('-a', '--all', action='store_true', help='Copy all files instead of only rst files.')
```

Comment:
Technically, it copies all the .rst files, PLUS files referenced by a configurable list of specific directives.

Reply:
Plus all other files that it finds in the folder actually. It will copy the scripts as well for example. Want me to fix the help text?

Reply:
Actualy, I was thinking of the behavior if the ``-a`` option is NOT given.

So the help text for ``-a`` could be:

    Copy all files (recursively) in the specified source folder(s).

and there should be some help information that explains what the script actually does

    usage: extract_content [-a, --all] <dest folder> <origin folder(s)>

    Recursively copy .rst files from the origin folder(s) to 
    the destination folder, plus files referenced in those .rst
    files by a configurable list of directives: <display the DIRECTIVES var here>